tests/components/application-connector/README.md

```diff
@@ -1,232 +1,5 @@
-
-## Debugging
-
-### Running locally
-
-> **CAUTION:** Because of the way it accesses the Application CRs, the test Job must run **on a cluster**.
-> Application Gateway and the mock application can both be run locally.
-
-To run the mock application locally, follow these steps:
-
-1. Change all the **targetUrl** values in the [Application CRs](./resources/charts/gateway-test/templates/applications/) to reflect the new application URL. For example, `http://localhost:8081/v1/api/unsecure/ok`.
-2. Change all the **centralGatewayUrl** values to reflect the new Application Gateway URL. For example, `http://localhost:8080/positive-authorisation/unsecure-always-ok`.
-3. Deploy all the resources on the cluster.
-   > **NOTE:** You can omit the test Job and the Central Gateway, but it's easier to just let them fail.
-4. Build the mock application:
-   
-   <div tabs name="Mock App Build Flavor" group="mock-app-flavor">
-   <details open>
-   <summary label="dockerized">
-   Docker
-   </summary>
-
-   ```shell
-   export DOCKER_TAG="local"
-   export DOCKER_PUSH_REPOSITORY="{DOCKER_USERNAME}"
-   make image-mock-app
-   ```
-
-   </details>
-   <details>
-   <summary label="local">
-   Local
-   </summary>
-
-   Change the hardcoded application port in [`config.go`](./tools/external-api-mock-app/config.go), and run:
-   ```shell
-   go build ./tools/external-api-mock-app/
-   ```
-   </details>
-   </div>
-5. Run the mock application:
-   
-   <div tabs name="Mock App Run Flavor" group="mock-app-flavor">
-   <details open>
-   <summary label="dockerized">
-   Docker
-   </summary>
-
-   ```shell
-   docker run -p 8180:8080 -p 8190:8090 -v "$PWD/k8s/gateway-test/certs:/etc/secret-volume:ro" "$DOCKER_PUSH_REPOSITORY/mock-app:$DOCKER_TAG"
-   ```
-
-   </details>
-   <details>
-   <summary label="local">
-   Local
-   </summary>
-
-   ```shell
-   ./external-api-mock-app
-   ```
-   > **CAUTION:** For the certificates to work, you must copy them from `./k8s/gateway-test/certs` to `/etc/secret-volume`.
-
-   </details>
-   </div>
-6. Run [Application Gateway](https://github.com/kyma-project/kyma/tree/main/components/central-application-gateway) with the `-kubeConfig {PATH_TO_YOUR_KUBECONFIG_FILE}` parameter.
-
-You can now send requests to Application Gateway, and debug its behavior locally.
-
-### Running without cleanup
-
-To run the tests without removing all the created resources afterwards, run them in the debugging mode.
-
-1. To start the tests in the debugging mode, run:
-
-   ``` shell
-   make disable-sidecar-for-mtls-test test-gateway-debug
-   ```
-
-2. Once you've finished debugging, run:
-
-   ``` shell
-   make clean-gateway-test enable-sidecar-after-mtls-test
-   ```
\ No newline at end of file
+There are the following component tests for Application Connector:
+- [Application Gateway](./application-gateway-tests.md)
+- [Connectivity Validator](./application-connectivity-validator-tests.md)
```

Comment:
As the `tests` folder has a structure of a repo, I'd further suggest moving these two files, as well as the `assets` folder, into a separate `docs` folder, not to clutter the main `tests` directory.